mne/tests/test_label.py

```diff
@@ -13,15 +13,17 @@
                            assert_equal)
 import pytest
 
+import mne
```

Comment:
we don't `import mne` in the test files; just import the specific functions you need